torch/_inductor/fx_passes/quantization.py

```diff
@@ -18,6 +18,15 @@
 quantized_decomposed = torch.ops.quantized_decomposed
 quantized = torch.ops.quantized
 
+
```

Comment:
could you add some docs about how bfloat16 is handled (and its interaction with quantization)?

Reply:
Thanks for the comments. The docs added and also the link to RFC for detail design. I will also update the tutorial for the usage of this feature.